ext/mysql2/statement.c

```diff
@@ -416,10 +421,16 @@ static VALUE rb_mysql_stmt_execute(int argc, VALUE *argv, VALUE self) {
     return Qnil;
   }
 
+  // Important to duplicate the hash, will receive merge! if extra opts
```

Comment:
Might be possible to duplicate only conditionally, or use `merge` instead?

Reply:
It was duplicate previously - I should actually check if it gets duplicated again in `rb_mysql_result_to_obj` or if that method keeps the same options hash argument that was passed to it.

Reply:
Confirmed that `rb_mysql_result_to_obj` uses what is passed to it, rather than duplicating its own query options hash, so the comment here (and in client.c) should reflect that.